data/brands/amenity/fast_food.json

```diff
@@ -5128,18 +5128,41 @@
       "displayName": "TKK Fried Chicken",
       "id": "tkkfriedchicken-571996",
       "locationSet": {
-        "include": ["cn", "tw", "us"]
+        "include": ["us"]
       },
       "tags": {
         "amenity": "fast_food",
         "brand": "TKK Fried Chicken",
+        "brand:en": "TKK Fried Chicken",
         "brand:wikidata": "Q15715284",
         "brand:wikipedia": "en:TKK Fried Chicken",
+        "brand:zh": "頂呱呱",
         "cuisine": "chicken",
         "name": "TKK Fried Chicken",
+        "name:en": "TKK Fried Chicken",
+        "name:zh": "頂呱呱",
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "頂呱呱",
```

Comment:
I'd prefer to split this item to 2 version in S/T Chinese
Sorry for waiting a while